test/test.py

```diff
@@ -143,83 +146,91 @@ def test_full_srdf(self):
   </link_sphere_approximation>
 </robot>
         '''
-        robot = SRDF.from_xml_string(srdf_data)
-        self.assertTrue( xml_matches(robot.to_xml_string(),expected))
-        
-  def test_simple_srdf(self):
-        datadir=rospkg.RosPack().get_path('srdfdom')+"/test/resources/"
-        stream = open(datadir+'pr2_desc.1.srdf', 'r')
-        robot = SRDF.from_xml_string(stream.read())
-        stream.close()
-        self.assertTrue(len(robot.virtual_joints)==0)
-        self.assertTrue(len(robot.groups)==0)
-        self.assertTrue(len(robot.group_states)==0)
-        self.assertTrue(len(robot.disable_collisionss)==0)
-        self.assertTrue(len(robot.end_effectors)==0)
-        
-        stream = open(datadir+'pr2_desc.2.srdf', 'r')
-        robot = SRDF.from_xml_string(stream.read())
-        stream.close()
-        self.assertTrue(len(robot.virtual_joints)==1)
-        self.assertTrue(len(robot.groups)==1)
-        self.assertTrue(len(robot.group_states)==0)
-        self.assertTrue(len(robot.disable_collisionss)==0)
-        self.assertTrue(len(robot.end_effectors)==0)
-        
+    robot = SRDF.from_xml_string(srdf_data)
+    self.assertTrue( xml_matches(robot.to_xml_string(False),expected))
+
+
+  # def test_simple_srdf(self):
+  #   _, package_path = get_resource("packages", PKG)
+  #   fname = os.path.join(
+  #       package_path, "share", PKG, "resources", "pr2_desc.1.srdf")
+  #   stream = open(fname, 'r')
+  #   robot = SRDF.from_xml_string(stream.read())
+  #   stream.close()
+  #   self.assertTrue(len(robot.virtual_joints)==0)
+  #   self.assertTrue(len(robot.groups)==0)
+  #   self.assertTrue(len(robot.group_states)==0)
+  #   self.assertTrue(len(robot.disable_collisionss)==0)
+  #   self.assertTrue(len(robot.end_effectors)==0)
+
+  def test_medium_srdf(self):
+    _, package_path = get_resource("packages", PKG)
+    fname = os.path.join(
+        package_path, "share", PKG, "resources", "pr2_desc.2.srdf")
+    stream = open(fname, 'r')
+    robot = SRDF.from_xml_string(stream.read())
+    stream.close()
+    self.assertTrue(len(robot.virtual_joints)==1)
+    self.assertTrue(len(robot.groups)==1)
+    self.assertTrue(len(robot.group_states)==0)
+    self.assertTrue(len(robot.disable_collisionss)==0)
+    self.assertTrue(len(robot.end_effectors)==0)
+
   def test_complex_srdf(self):
-        datadir=rospkg.RosPack().get_path('srdfdom')+"/test/resources/"
```

Comment:
This file contains a lot of unnecessary reformatting.

Reply:
I believe the reformatting is necessary as it is bad style to change indentation mid file. I have refactored my changes so that the whitespace changes are in a separate commit

Reply:
@rhaschke is right, this should be in a separate PR